bundles/config/org.eclipse.smarthome.config.core.test/META-INF/MANIFEST.MF

```diff
@@ -12,5 +12,5 @@ Import-Package: groovy.lang,
  org.codehaus.groovy.runtime.callsite,
  org.codehaus.groovy.runtime.typehandling,
  org.eclipse.smarthome.test,
- org.hamcrest;core=split
-Require-Bundle: org.junit;bundle-version="4.0.0"
+ org.hamcrest;core=split,
+ org.junit;version="4.5.0"
```

Comment:
At the moment there are different versions used:
* 3? (junit.framework)
* 4.0.0
* 4.5.0
* 4.8.1

So, I have to choose one.
Do you prefer 4.0.0?
What is with the bindings currently set 4.5.0 or 4.8.1?